services/gss2.py

```diff
@@ -48,12 +48,12 @@ def plugin(srv, item):
         srv.logging.debug("Adding row to spreadsheet %s [%s]..." % (spreadsheet_url, worksheet_name))
         if os.path.isfile(oauth2_storage_filename):
             # Valid credentials from previously completed authentication?
-            srv.logging.debug(u"Trying to use credentials from file '%s'." % 
+            srv.logging.debug(u"Trying to use credentials from file '%s'." %
```

Comment:
Are these actual changes?

Reply:
I opened the file using github desktop and atom and it auto removed the spaces in the file. See the missing spaces. Those were the only changes were deletion of extra spaces.